clusters/app.ci/prow-monitoring/mixins/prometheus_out/_alertmanager.yaml

```diff
@@ -98,5 +101,8 @@ route:
   - match:
       severity: critical
     receiver: slack-criticals
+  - match:
+      severity: critical
+    receiver: pager-duty
```

Comment:
Do you want to the alerts with `severity: critical` go to both slack AND pager-duty?
I would do it like https://www.robustperception.io/sending-alert-notifications-to-multiple-destinations
Add `pagerduty_configs` into the existing receiver `slack-criticals`.
Otherwise, we have to think about `continue: true`.